include/deal.II/lac/petsc_solver.h

```diff
@@ -163,7 +163,10 @@ namespace PETScWrappers
     void initialize(const PreconditionerBase &preconditioner);
 
     /**
-     * Exception
+     * Exception.
+     *
+     * @deprecated This function has been deprecated in favor of the more
+     * general LACExceptions::ExcPETScError exception class.
      */
     DeclException1 (ExcPETScError,
```

Comment:
Can you put `DEAL_II_DEPRECATED` at the end of this declaration, or does this not conform to the proper syntax?